wolfssl/ssl.h

```diff
@@ -2377,65 +2377,6 @@ enum {
                               */
     SSL_MODE_RELEASE_BUFFERS = -1, /* For libwebsockets build. No current use. */
 
-    /* Not all of these are actually used in wolfSSL. Some are included to
```

Comment:
The enums are should be visible in `wolfssl/openss/ssl.h`, since that file includes `wolfssl/ssl.h`. Maybe not all `#ifdef' conditions are met? Do we need to add some `defined(WOLFSSL_HAPROXY)` or others here?

Reply:
@icing, HAProxy CLI [(ssl_util.c)](https://github.com/haproxy/haproxy/blob/master/src/ssl_utils.c#L433) expects them to be defines. I think the `OPENSSL_EXTRA` guard should be sufficient. Let me know if I missed something?